### DISCUSSION TITLE
Wrong error message in CMakeLists.txt file of LxQT_Panel repository?

### DISCUSSION THREAD

**Discussion by thiyashoksanju** [lxqt/lxqt#2232] [General]
- i was trying to compile and build the lxqt panel repository. While doing that i got few errors stating that some dependent libraries are missing.
- By referring the error messages i installed the required libraries and continued the build process.
- One error message shown like "**Sensors plugin requires lm_sensors**". So i installed lm-sensors package. Still the error exists.
- Went through the CMakeLists.txt file of lxqt_panel repository, it has the line of **find_library(SENSORS_LIB sensors) (Line 156)**. It requires libsensors but the error message says lm-sensors.
- So i removed the previously installed lm-sensors and installed "**libsensors-dev**". Now the error is resolved.
- The error message (**line 159**) did lead to a wrong installation and stuck at the compilation process. Its better to verify this statement and change the error message if required
- Repository details

1.  https://github.com/lxqt/lxqt-panel
2. version 1.1.0
3. Lubuntu version 22.04 (running as VM)

**Comment by kalgnana** [comment]
It would be good if fix this wrong message issue.

**Comment by palinek** [comment]
https://github.com/lxqt/lxqt-panel/pull/1782
